src/elm/Route/Path.elm

```diff
@@ -11,6 +11,8 @@ import Url exposing (Url)
 import Url.Parser exposing ((</>))
 
```

Comment:
i think this whole file will get overwritten by `elm-land generate`, should we remove the comments?

Reply:
comments removed.